sdk/nodejs/provisioning/test/provisioner.spec.ts

```diff
@@ -0,0 +1,19 @@
+import assert from 'assert';
+import { getProvisioner } from '../provisioner.js';
+import { DEFAULT_HOST } from '../default.js';
+
+describe('Provisioner', function () {
+	describe('Docker Image', function () {
+		it('Should create a GQL client on call to connect', async function () {
```

Comment:
As a follow up it would be good to modify this test case to be similar to the one in Go or Python and run provisioners in parallel with each other: https://github.com/dagger/dagger/blob/main/sdk/python/tests/connectors/test_docker.py#L18

Not a blocker for merge though